core/vm/eips.go

```diff
@@ -46,9 +45,9 @@ func EnableEIP(eipNum int, jt *JumpTable) error {
 // - Define SELFBALANCE, with cost GasFastStep (5)
 func enable1884(jt *JumpTable) {
 	// Gas cost changes
-	jt[BALANCE].constantGas = params.BalanceGasEIP1884
-	jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
-	jt[SLOAD].constantGas = params.SloadGasEIP1884
+	// jt[BALANCE].constantGas = params.BalanceGasEIP1884
+	// jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
```

Comment:
We should include these changes unless they also break things. My understanding is that they don't.

Reply:
Well they might break stuff similarly to how SLOAD does, but probably not in the case we detected (transfer to governance).

Reply:
Good call. I agree being on the safe side of this is probably better. 👌